src/gluonnlp/model/bert.py

```diff
@@ -602,6 +604,174 @@ def __call__(self, inputs, valid_length=None, masked_positions=None):
+    prefix : str or None
+        See document of `mx.gluon.Block`.
+    params : ParameterDict or None
+        See document of `mx.gluon.Block`.
+    """
+
+    def __init__(self, bert, num_classes=2, dropout=0.0,
+                 prefix=None, params=None):
+        super(BERTClassifier, self).__init__(prefix=prefix, params=params)
+        self.bert = bert
+        with self.name_scope():
+            self.classifier = nn.HybridSequential(prefix=prefix)
+            if dropout:
+                self.classifier.add(nn.Dropout(rate=dropout))
+            self.classifier.add(nn.Dense(units=num_classes))
+
+    def __call__(self, inputs, token_types, valid_length=None):
+        # pylint: disable=dangerous-default-value, arguments-differ
+        """Generate the unnormalized score for the given the input sequences.
+
+        Parameters
+        ----------
+        inputs : NDArray or Symbol, shape (batch_size, seq_length)
+            Input words for the sequences.
+        token_types : NDArray or Symbol, shape (batch_size, seq_length)
+            Token types for the sequences, used to indicate whether the word belongs to the
+            first sentence or the second one.
+        valid_length : NDArray or Symbol, or None, shape (batch_size)
+            Valid length of the sequence. This is used to mask the padded tokens.
+
+        Returns
+        -------
+        outputs : NDArray or Symbol
+            Shape (batch_size, num_classes)
+        """
+        # XXX Temporary hack for hybridization as hybridblock does not support None inputs
+        valid_length = [] if valid_length is None else valid_length
+        return super(BERTClassifier, self).__call__(inputs, token_types, valid_length)
+
+    def hybrid_forward(self, F, inputs, token_types, valid_length=None):
+        # pylint: disable=arguments-differ
+        """Generate the unnormalized score for the given the input sequences.
+
+        Parameters
+        ----------
+        inputs : NDArray or Symbol, shape (batch_size, seq_length)
+            Input words for the sequences.
+        token_types : NDArray or Symbol, shape (batch_size, seq_length)
+            Token types for the sequences, used to indicate whether the word belongs to the
+            first sentence or the second one.
+        valid_length : NDArray or None, shape (batch_size)
+            Valid length of the sequence. This is used to mask the padded tokens.
+
+        Returns
+        -------
+        outputs : NDArray
+            Shape (batch_size, num_classes)
+        """
+        # XXX Temporary hack for hybridization as hybridblock does not support None
+        if isinstance(valid_length, list) and len(valid_length) == 0:
+            valid_length = None
+        _, pooler_out = self.bert(inputs, token_types, valid_length)
+        return self.classifier(pooler_out)
+
+class RoBERTaClassifier(HybridBlock):
+    """Model for sentence (pair) classification task with BERT.
```

Comment:
The documentation might need be updated that RoBERTa model doesn't have token_types as input

Reply:
I didn't include token_types in the documentation for RoBERTa. Where did you see `token_types `?

Reply:
@eric-haibin-lin Oh, Oh, nvm. That is a mistake